65black/so-lite-react
# #162832556 Add Get All Questions Functionality

#### What does this PR do?
This PR adds the ability to fetch all questions by navigating to a URL.

#### Description of Task to be completed?
- add ability to get all questions
- add tests for new feature
- update package.json for heroku

#### How should this be manually tested?
- Clone the repo, run `npm install` and run `npm start`
- Navigate to `localhost:9000/questions`
- You should see a list of all the questions on StackO-lite

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162832556

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-12-27 at 2 43 09 am" src="https://user-images.githubusercontent.com/36507972/50461779-29e7d080-0981-11e9-8f88-28fa33571852.png">


#### Questions:
N/A
